src/components/Avatar/Avatar.tsx

```diff
@@ -53,7 +53,28 @@ export interface Props {
   variant?: 'icon' | 'initials' | 'image';
 }
 
-function getInitials(fromName: string): string {
+/**
+ * Algo for names with special characters in:
+ * - Split name parts by space
+ * - for a given part,
+ *  - if that part has emoji or extended characters, {Action?}
```

Comment:
Is `{Action?}` a placeholder for a future edit to the comment?

Reply:
Yep. Gonna clean this up after digging in some more later today